app/models/message.rb

```diff
@@ -0,0 +1,8 @@
+class Message < ActiveRecord::Base
+  belongs_to :conversation
+  belongs_to :user
+
+  validates_presence_of :body
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.